resource_quota.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/client-go/1.5/pkg/api/v1"
+)
+
+var (
+	descResourceQuota = prometheus.NewDesc(
+		"kube_resource_quota",
+		"Information about resource quota.",
+		[]string{
+			"name",
```

Comment:
sorry I seem to have missed this one earlier, this should be `resourcequota` instead of `name`, for consistency with the other namings as then joining labels in prometheus is much easier